Makefile

```diff
@@ -19,18 +19,28 @@ node_modules: package.json package-lock.json
 core/css/uikit.%.css: src/themes/%.less node_modules
 	node_modules/less/bin/lessc src/themes/$*.less core/css/uikit.$*.css --relative-urls
 
+core/css/material-icons.css: src/themes/icons.scss node_modules
+	node node_modules/.bin/sass src/themes/icons.scss core/css/material-icons.css --no-source-map
+
+core/fonts: node_modules
+	mkdir core/fonts
+
+core/fonts/MaterialIcons-Regular.%: core/fonts node_modules
+	cp node_modules/material-icons/iconfont/MaterialIcons-Regular.* core/fonts/
```

Comment:
if you use the placeholder `%` in the rule name, you need to also use it somewhere here in this command as `$*`.

Reply:
well - I basically want all MaterialIcons-Regular.* to be copied over

what would be the best rule to write this?

_still asking myself if this is the right approach - can't webpack just grab that?_

Reply:
yeah, I suspect webpack should be able to grab and even compile some stuff directly without needing extra Makefile rules.